packages/govuk-frontend/src/govuk/errors/index.mjs

```diff
@@ -48,23 +48,18 @@ export class ElementError extends GOVUKFrontendError {
   name = 'ElementError'
 
   /**
-   * @param {Element | null} element - The element in error
+   * @param {string} identifier - An identifier that'll let the user understand which element has an error (variable name, CSS selector)
    * @param {object} options - Element error options
    * @param {string} options.componentName - The name of the component throwing the error
-   * @param {string} options.identifier - An identifier that'll let the user understand which element has an error (variable name, CSS selector)
-   * @param {string | typeof HTMLElement} [options.expectedType] - The type that was expected for the identifier
+   * @param {Element} [options.element] - The element in error
```

Comment:
Mind if we keep your `Element | null` change from https://github.com/alphagov/govuk-frontend/pull/4263/commits/1377822a64c3b3eb92d32290f3edacc76bf9dc21?

Selectors `document.querySelector()` and `document.getElementById()` return `null` when not found 

```suggestion
   * @param {Element | null} [options.element] - The element in error
```